velox/functions/sparksql/GetJsonObject.h

```diff
@@ -116,6 +118,22 @@ struct GetJsonObjectFunction {
     return result;
   }
 
+  // Normalizes the JSON path to be Spark-compatible:
+  // - Removes single quotes in bracket notation
+  // - Removes spaces after dots (e.g., "$. a" -> "$.a")
+  std::string normalizeJsonPath(StringView jsonPath) {
+    // First, remove single quotes for bracket notation
+    const std::string& path = removeSingleQuotes(jsonPath);
+    if (path == "-1") {
+      return path;
+    }
+
+    // Use Boost regex to find and remove spaces after dots
+    // Pattern: "dot + one or more spaces" -> "dot"
+    static const boost::regex dotSpaceRegex("\\.\\s+");
+    return boost::regex_replace(path, dotSpaceRegex, ".");
```

Comment:
@nimesh1601, please consider the nested json structure in the test. For example, we have a test: `EXPECT_EQ(getJsonObject(R"({"my": {"hello": 3.5}})", "$.my.hello"), "3.5");`, what if json path is `"$.  my.    hello"`? Please confirm Spark's result.